.github/workflows/deploy.yml

```diff
@@ -2,20 +2,54 @@ name: Deploy Docker Image
 
 on:
   push:
-    tags:
-      - v*
+    branches:
+      - master
+      - main
```

Comment:
I don't know if deploy on every master / main push is a good idea. I prefer doing in on tag but I may be the one wrong here :o

Reply:
yeah about that,
i just deleted this file